src/tests/Microsoft.PowerFx.Core.Tests.Shared/UserDefinedFunctionTests.cs

```diff
@@ -575,6 +575,25 @@ public void TestUDFsReservedNamesTracking()
                     Assert.Contains(errors, x => x.MessageKey == "ErrUDF_FunctionNameRestricted");
                 }
             }
+        }
+
+        [Theory]
+        [InlineData("func():Void { Set(x, 123) };")]
+        [InlineData("func():Void { Set(x, 123); Set(y, 123) };")]
+        [InlineData("func():Void = { Set(x, 123) };")]
+        [InlineData("func():Void = { Set(x, 123); Set(y, 123) };")]
+        public void TestImperativeUDFParseWithoutSemicolon(string script)
```

Comment:
>TestImperativeUDFParseWithoutSemicolon [](http://example.com/codeflow?start=20&length=38)

Not sure if I understood. The test name state we are testing an UDF without semicolon but what differ between them is the equal sign.

Reply:
Before this change, we would need a semicolon after the expression like this `Set(x, 123);`, if we didn't have an equal sign. This test is ensuring parity between equal sign or no.

Reply:
I see. Thanks for explaining.